### DISCUSSION TITLE
Dual Temp Display or Toggle switch?

### DISCUSSION THREAD

**Discussion by ChrisM510** [poblabs/weewx-belchertown#803] [Ideas]
I was wondering if I am the only one that would like to see temps in both Fahrenheit and Celsius...  My thought was to have one as the primary value, and then another as a smaller font value nearby, or perhaps by having a toggle switch like the light/dark theme.  Thoughts?

**Comment by ThePeePs** [comment]
Interesting idea. Might need to do some math in JS for the MQTT updates, as (looking at the code) it seems to be either metric or imperial.  Having it as a toggle would be easier layout wise, as it would just be replacing the values (like how the forecast is done)  Were you thinking for just the number for the temps, or for the charts as well? (and for that matter, for all values displayed?)

**Reply by ChrisM510** [reply]
I was just thinking of the temperature... I work with a lot of Canadians and Europeans via Zoom, and often the weather comes up before a meeting. LOL

Other values would be cool, one day, but I wasn't looking to boil the ocean...yet. 😉
